### PR TITLE
Fe file uploads

### PR DESCRIPTION
**Pull request by davida26**
Footer updates
Target pathways connected
Uploader
Endpoint tweaks
KG tweaks

### CODE REVIEW

File: backend/search/csv_service.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
+from django.http import HttpResponse
+from neomodel import db
+
+URI = "bolt://localhost:7687"
+AUTH = ("neo4j", "gradvek1")
```
Comment (nsheely) [inline backend/search/csv_service.py]: Shouldn't have URI and AUTH here, all the database login is handled in startup.py. Auth info is stored in .env or environment variables.

Should just be able to use the neomodel db functions as long as startup has happened.

**Review by nsheely — CHANGES_REQUESTED** [review]

File: backend/search/csv_service.py (hunk 1 above)
Comment (davida26) [reply backend/search/csv_service.py]: @kevinpatel24 - this merge includes your CSV branch could you take a look at @nsheely's comment

Comment (kevinpatel24) [reply backend/search/csv_service.py]: Just made a push to the fe-file-uploads branch that removes those two lines. I tested without them and the path still works as Nathan said they would

**Review by kevinpatel24 — APPROVED** [review]
Removed those lines in the latest commit

**Review by nsheely — APPROVED** [review]

**Pull request merged**